website/addons/badges/templates/badges_user_settings.mako

```diff
@@ -96,6 +96,7 @@ You have not created any Badges.
 
 //todo fetch url from server
   var jsonToList = function(badge, id) {
+      // TODO: Rewrite this substantially before this feature is re-activated
```

Comment:
This could use some explanation for future developer archaeologists. Maybe a link to the markup safe issue or a comment about what needs re-writing?